### DISCUSSION TITLE
Not able to launch Unreal Tournament on my M2 Mac all of a sudden

### DISCUSSION THREAD

**Discussion by deathwinger** [OldUnreal/UnrealTournamentPatches#1575] [General]
Hey there, I was able to initially play Unreal Tournament on my M2 Mac as recently as a month ago. However, I think since its been updated to  Sonama 14.4.1 (23E224), the application is not launching anymore. 

When I click on it, it just jumps up and down but then nothing happens. I've tried restarting, and I've tried starting it multiple times. Nothing has worked. 

Is anyone else getting this issue recently, and if so, how do I fix it?

Thanks for any assistance anyone can provide.

**Comment by SeriousBuggie** [comment]
Did you update to latest 469d image for Sonoma?
https://github.com/OldUnreal/UnrealTournamentPatches/releases/latest
`OldUnreal-UTPatch469d-macOS-Sonoma.dmg`

**Comment by deathwinger** [comment]
This worked. I didn’t realize there was that recent update just for Mac
Sonoma. Thank you very much.

Best.

Mikkell K. Khan
Author. Digital Marketer. Digital Producer.
“I Write Stories For Readers To Get Lost In and Learn From”
***@***.***
www.mikkellkhan.com
